pkg/cluster/spec/instance.go

```diff
@@ -51,6 +51,7 @@ const (
 	ComponentPump             = "pump"
 	ComponentCDC              = "cdc"
 	ComponentTiKVCDC          = "tikv-cdc"
+	ComponentTiCI             = "tici"
```

Comment:
Why do we still need `ComponentTiCI` while we have `ComponentTiCIMeta` and `ComponentTiCIWorker`?

Reply:
`ComponentTiCI` is used to install binary.